Rethinking_2/Chp_02.ipynb

```diff
@@ -61,7 +61,7 @@
     "$$Pr(w \\mid n, p) =  \\frac{n!}{w!(n − w)!} p^w (1 − p)^{n−w}$$\n",
```

Comment:
<p>It seems like you didn't run this cell -- the plot is no longer there.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>